protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -312,6 +314,15 @@ message TransactionPayload {
     MultisigPayload multisig_payload = 6;
   }
   reserved 4;
+
+  oneof extra_config {
+    ExtraConfigV1 extra_config_v1 = 7;
+  }
+}
+
+message ExtraConfigV1 {
+  optional string multisig_address = 1;
+  optional uint64 replay_protection_nonce = 2;
```

Comment:
remove the `optional` if it is not optional?

Reply:
Those fields are optional. 
`multisig_address` is present only for multisig transactions.
`replay_protection_nonce` is present only for orderless transactions.

Reply:
ummm.. shall we put the nonce together with the sequence number and make them an oneof?

Reply:
Currently, I only added ExtraConfig field in the transaction payload but didn't change the type of any existing field to make sure nothing breaks. Is changing sequence number to replay protector enum (sequence number/nonce) backward compatible?
cc @bowenyang007 @rtso

Reply:
changing to enum is not compatible, to oneof is (as long as you don't change the tag number).